8/readme.md

```diff
@@ -1,41 +1,63 @@
 # readme
 
-This example builds on top of solution 6 and demonstrates how to:
-
-* control grains features using the [Feature Management Library](https://www.nuget.org/packages/Microsoft.FeatureManagement/)
-* unit test grains using [OrleansTestKit](https://www.nuget.org/packages/OrleansTestKit/)
-* do dependency injection in grains
+This example is somewhat different to previous examples as it's hosted in ECS.
+* A membership table is created in Dynamo db, this eliminates the need for a Primary Silo endpoint as endpoint details are stored in dynamo. 
+It also means that the client only needs the dynamo connection info to find all available silos.
+* Extension methods have also been created for building silos to abstract setup.
+* A CloudFormation template has also been provided to instantiate the sample in AWS.
+* Demonstrations of local instances connecting to cloud based clusters are also included.
 
 ## running the code
+### ECS
+* To pull this code in CloudFormation, it'll be necessary to store your github user and authentication token in the aws secrets manager
+ so the image can be pulled from git container registry. Then point at the secret in the cloud formation template, 
+ below is a sample of how the secret should be stored.
+
+`{
+  "username" : "mrblonde91",
+  "password" : "token"
+}`
+* Run the template to pull the image and it will start creating the containers.
+
+![Container ports info](imgs/1.png)
 
-### silos
 
-As in example 7, following scripts can be used to run silos:
+* Internally the ports are defaulted and externally an available one is selected.
+![Dynamo table info](imgs/2.png)
+* A sample of the dynamo data is displayed above, the cluster id acts as a unique identifier for each cluster. So multiple clusters can be stored 
+in the same table if the user wishes. Clusters can also contain different versions of silos eg for testing purposes.
+### Running Silo local -> Aws Cluster
```

Comment:
I think this could be made clearer: `running local and ECS-hosted silos in one cluster`? Or something better phrased.

Reply:
Done